libs/utils/utils.js

```diff
@@ -1148,6 +1148,7 @@ function decorateMeta() {
   const contents = document.head.querySelectorAll(`[content*=".${SLD}."]`);
   contents.forEach((meta) => {
     if (meta.getAttribute('property') === 'hlx:proxyUrl') return;
+    if (meta.getAttribute('name').endsWith('schedule')) return;
```

Comment:
Can we put this if logic with an OR in the if condition on line number 1150 like below

`if (meta.getAttribute('property') === 'hlx:proxyUrl' || meta.getAttribute('name').endsWith('schedule')) return;`